packages/orderbook/src/orderbook.ts

```diff
@@ -84,6 +84,26 @@ export class Orderbook {
             o => o.order.makerAssetData === makerAssetData && o.order.takerAssetData === takerAssetData,
         );
     }
+    public async getBatchOrdersAsync(makerAssetDatas: string[], takerAssetDatas: string[]): Promise<APIOrder[][]> {
+        for (const [mi, makerAssetData] of makerAssetDatas.entries()) {
```

Comment:
Would it be better to explicitly pass in pairs instead of generating combinations to avoid doing more work than necessary?

Reply:
mm yeah could be a good idea. Let me think on this